glossary/QUNIT.md

```diff
@@ -0,0 +1,3 @@
+# QUnit
+
+[QUnit](http://api.qunitjs.com) section to be completed.
```

Comment:
You can link to https instead of http.

The https://api.qunitjs.com if for the API docs, but the main website is only https://qunitjs.com